datalab/mlalpha/commands/_mlalpha.py

```diff
@@ -997,10 +1012,17 @@ def _feature_slice_view(args, cell):
 browser.weightedExamplesColumn = 'totalWeightedExamples';
 browser.calibrationPlotUriFn = function(s) { return '/' + s; }
 </script>"""
-  with open(args['file']) as f:
-    data = map(json.loads, f)
-  if args['feature']:
-    data = [e for e in data if e['feature'].split(':')[0] == args['feature']]
+  if args['sql'] is not None:
+    item = datalab.utils.commands.get_notebook_item(args['sql'])
+    item, _ = datalab.data.SqlModule.get_sql_statement_with_environment(item, {})
+    query = datalab.bigquery.Query(item)
+    df = query.results().to_dataframe()
+    data = _get_lantern_format(df)
+  elif args['dataframe'] is not None:
+    item = datalab.utils.commands.get_notebook_item(args['sql'])
```

Comment:
args['sql'] -> args['dataframe'] ???

I'm not sure how testing works on github, but if this is an error, does it mean we are missing a test case?